grails-app/utils/org/broadinstitute/orsp/utils/UtilityClass.groovy

```diff
@@ -150,6 +152,23 @@ class UtilityClass {
         }
     }
 
+    void registerIssueListMarshaller() {
+        JSON.createNamedConfig(ISSUE_LIST) {
+            SimpleDateFormat  sd = new SimpleDateFormat("yyyy-MM-dd")
+            it.registerObjectMarshaller( Issue ) { Issue ie ->
+                return [
+                        id               : ie.id,
+                        projectKey       : ie.projectKey,
+                        summary          : IssueUtils.escapeQuote(ie.summary),
+                        status           : IssueUtils.escapeQuote(ie.status),
+                        type             : IssueUtils.escapeQuote(ie.type),
+                        updateDate       : ie.updateDate ? sd.format(ie.updateDate) : '',
+                        actors           : queryService.findUsersInUserNameList((List<String>)ie.getActorUsernames())?.collect { it.displayName }
```

Comment:
Why the need for casting here `(List<String>)ie.getActorUsernames()` ? Issue already returns a `Collection<String>`